map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -74,24 +76,29 @@
   @FXML
   private void initialize() {
 
-    // set basemap options
-    basemapList.getItems().addAll(Basemap.createStreets(), Basemap.createImagery(), Basemap
-            .createTopographic(), Basemap.createOceans());
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-    // update basemap when selection changes
-    basemapList.getSelectionModel().select(0);
-    basemapList.getSelectionModel().selectedItemProperty()
-            .addListener(o -> map.setBasemap(basemapList.getSelectionModel().getSelectedItem()));
+    // set the basemap style options
+    basemapStyleList.getItems().addAll(BasemapStyle.ARCGIS_STREETS, BasemapStyle.ARCGIS_IMAGERY_STANDARD,
+      BasemapStyle.ARCGIS_TOPOGRAPHIC, BasemapStyle.ARCGIS_OCEANS);
 
-    basemapList.setCellFactory(c -> new BasemapCell());
+    // update the basemap when the selection changes
+    basemapStyleList.getSelectionModel().select(0);
+    basemapStyleList.getSelectionModel().selectedItemProperty()
+      .addListener(o -> map.setBasemap(new Basemap(basemapStyleList.getSelectionModel().getSelectedItem())));
 
-    // create and set a map with the first basemap option
-    map = new ArcGISMap(basemapList.getSelectionModel().getSelectedItem());
+    basemapStyleList.setCellFactory(c -> new BasemapCell());
```

Comment:
Perhaps it'd be easier to do this after line 85, to do all the setup steps for the list view at once (i.e. put in the values, set the cell factory)